app/service-catalog/services.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Service Catalog services"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+    - service-catalog
+works_on:
+  - konnect
+
+description: Learn about services in Service Catalog and how to configure them.
+
+breadcrumbs:
+  - /service-catalog/
+
+related_resources:
+  - text: "Service Catalog"
+    url: /service-catalog/
+  - text: Scorecards
+    url: /service-catalog/scorecards/
+  - text: Traceable integration
+    url: /service-catalog/integrations/traceable/
+  - text: GitHub integration
+    url: /service-catalog/integrations/github/
+  - text: GitLab integration
+    url: /service-catalog/integrations/gitlab/
+  - text: SwaggerHub integration
+    url: /service-catalog/integrations/swaggerhub/
+  - text: Datadog integration
+    url: /service-catalog/integrations/datadog/
+  - text: PagerDuty integration
+    url: /service-catalog/integrations/pagerduty/
+---
```

Comment:
I think it would be helpful to add an FAQ here that says something like "What's the different between a Gateway Service and a Service Catalog service?". We even have definitions for both in our glossary: https://developer.konghq.com/glossary/